compiler/docgen.nim

```diff
@@ -222,6 +222,10 @@ proc getPlainDocstring(n: PNode): string =
       result = getPlainDocstring(n.sons[i])
       if result.len > 0: return
 
+func htmlCollapse(html: string): string =
```

Comment:
Don't use ``func``, it's too new. Also it's used nowhere else in ``docgen.nim``.